packages/utils/src/common/sharding.spec.ts

```diff
@@ -91,10 +96,22 @@ describe("contentTopicToShardIndex", () => {
   it("converts content topics to expected shard index", () => {
     const contentTopics: [string, number][] = [
       ["/toychat/2/huilong/proto", 3],
-      ["/myapp/1/latest/proto", 0]
+      ["/myapp/1/latest/proto", 0],
+      ["/waku/2/content/test.js", 1]
     ];
     for (const [topic, shard] of contentTopics) {
       expect(contentTopicToShardIndex(topic)).to.eq(shard);
     }
   });
 });
+
```

Comment:
I would add a fast check here and confirm that any content topic with same app name are in the same shard.

Reply:
Added in latest commit

Reply:
> Added in latest commit

I don't see it.

I'd expect something using the `fast-check` package.

e.g.

```
import fc from "fast-check";

  it("Content topics for the same app go in the same shard", function () {
    fc.assert(
      fc.Property(
        fc.string({ minLength: 1 }), // app name
	fc.number(), // app version 1
        fc.string({ minLength: 1 }), // topic name 1
        fc.string({ minLength: 1 }), // encoding 1
	fc.number(), // app version 2 (I don't remember if this is used for autosharding)
        fc.string({ minLength: 1 }), // topic name 2
        fc.string({ minLength: 1 }), // encoding 2
        (appName, appVersion1, topicName1, encoding1, appVersion2, topicName2, encoding2) => {
          const contentTopic1 = `/${appName}/${appVersion1}/${topicName1}/${encoding1}`;
          const contentTopic2 = `/${appName}/${appVersion2}/${topicName2}/${encoding2}`;
	
	      expect(contentTopicToShardIndex(contentTopic1)).to.eq(contentTopicToShardIndex(contentTopic1));
})
})
```

Then do the same where appname are different